examples/spe1/render.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
```

Comment:
This entire script is a hack to get hold of the `template_render` job from `libres` as it is not installed into `PATH`.